src/storage/src/hummock/compactor/mod.rs

```diff
@@ -221,6 +222,7 @@ impl Compactor {
                 .verbose_instrument_await("upload")
                 .await
                 .map_err(HummockError::sstable_upload_error);
+            metrics.compaction_uploading_sst_counts.desc();
```

Comment:
Hmm... This metric doesn't measure `uploading_sst_count` but sst count for running task including uploaded and uploading SSTs because we inc the counter when the upload is initiated but desc the counter when the task finishes, not upload finishes.